ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -2725,9 +2725,10 @@ protected int doPrepare (boolean reportHeuristics, AbstractRecord record)
                     }
 
                     /*
-                              * Prepare on this record failed - we are in trouble.
-                              * Add the record back onto the pendingList and return.
-                              */
+                     * Prepare on this record failed - we are in trouble.
+                     * Add the record back onto the pendingList and return.
+                     */
+
                     addDeferredThrowables(record, deferredThrowables);
```

Comment:
The comment just above is now a little unclear . Move this call to before the comment.